src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -196,6 +209,20 @@ public static TypedData ToRpc(this object value, ILogger logger)
             return typedData;
         }
 
+        private static bool IsRawBodyBytesRequested(Capabilities capabilities)
+        {
+            return !string.Equals(capabilities.GetCapabilityState(LanguageWorkerConstants.RawHttpBodyBytes), null);
```

Comment:
change to `return capabilities.GetCapabilityState(LanguageWorkerConstants.RawHttpBodyBytes) != null;`